bot/internal/review/review.go

```diff
@@ -260,6 +268,21 @@ func (r *Assignments) getPreferredReviewers(set []string, files []github.PullReq
 	return preferredReviewers
 }
 
+// getAllPreferredReviewers returns a list of reviewers that would be
+// preferrable to review the provided changeset. Includes all preferred
+// reviewers for each file path in the chagne set.
+func (r *Assignments) getAllPreferredReviewers(set []string, files []github.PullRequestFile) (preferredReviewers []string) {
+	for path, reviewers := range r.getPreferredReviewersMap(set) {
+		for _, file := range files {
+			if !strings.HasPrefix(file.Name, path) {
+				continue
+			}
+			preferredReviewers = append(preferredReviewers, reviewers...)
```

Comment:
Feels like this can result in the same reviewer being added multiple times - is this okay?

Reply:
Oversight on my part! I've changed this to deduplicate `preferredReviewers`. The triple `for` loop isn't the greatest (it assumes the number of file paths and reviewers per path remains small), so I've explained the logic in a comment.